docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,130 @@
+# Faker connector
```

Comment:
Do we ever anticipate supporting other faker libraries in this connector? Should we rename it to datafaker connector? To follow the name of the upstream library? 

Also should we use the logo of datafaker here?

Reply:
The first Faker library was written in Perl, then ported to Ruby, and then to most other programming languages. Datafaker is not the only one available in Java, but I've chosen it since it's actively maintained, and attracts lots of contributions over the last 2 years.

Right now, there's no reason to replace it, but it's always an option. Supporting multiple ones will be confusing to users.

I chose to name the connector faker, since it's short, unique, easy to pronounce, type, and sets the right expectations for anyone that used these faker libraries in any language. Definitely better than `trino-random-generator` :-)

Reply:
Ok .. lets not use the datafaker log .. but still explain that the library is used for some important parts of the plugin in a secondary sentence on the intro.